docs/REQUIREMENTS.md

```diff
@@ -7,8 +7,92 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
+That includes being able to:
+   - Discover the image that fits the selected host.
+   - Find the best match from the nodes and images I have available.
+   - Determine that the image is not fitting the selected host.
+1. As a deployment engineer, I want the image compatibility check to be performed without downloading or executing the referenced image layers.
+1. As a deployment engineer, I should be able to add the compatibility to images already being used in production, especially for the images released before image compatibility wg was created.
+1. As a deployment engineer, I want to be able to specify the version and variant of an application or other user specific configuration (e.g. MPI), and not only hardware/kernel details in the compatibility specification.
+1. As a deployment engineer, I want my compatibility spec runtime to be able to select the best possible runtime available on a node (e.g. runc vs nvidia vs wasm).
+1. As a deployment engineer, I want to be able to rank the images in a multi-platform image so that the runtime can know which one to choose when more than one image is compatible with the runtime environment.
+1. As a deployment engineer, I want to reuse community projects so that I don't duplicate and integrate the functionality of the already existing tools.
+
+### Registry Maintainer
+
+1. As a registry user or operator I want to have a common way of inspecting information about image compatibility to enable users to find an image that best matches their system.
+1. As a registry operator (or user that has no control over their registry implementation), I want any compatibility changes to not depend on registry server changes or upgrades.
+
+### OCI Specification Maintainer
+
+1. As a spec maintainer, I want the solution to avoid breaking other specs (confidential images, image signing, existing implementations for runtimes picking images).
+1. As a spec maintainer, I don't want the spec to update for new hardware devices, kernel releases, or other external dependencies.
+1. As a spec maintainer, I don't want to overlap significantly with solutions from other specs (like SBOMs).
+
+### Security Administrator
+
+1. As a security administrator, I want predictable behavior from runtimes, which does not change based on unsigned content.
+1. As a security administrator, I want to ensure the compatibility spec cannot be used to escalate privileges beyond what is requested by the deployment engineer.
+1. As a security (or more generally, XYZ) researcher, I want to annotate a container (separately) with my niche jargon of metadata.
+1. As a security administrator, I want to know that image compatibility cannot be used to circumvent image signing.
+1. As a security administrator I want to catalog SBOMs of compatible images to put into a report about software used by my group / institution.
```

Comment:
should the raw requirements be listed here. In wg-reference-types the requirements document includes raw requirements, also credit to contributors: https://github.com/opencontainers/wg-reference-types/blob/main/docs/REQUIREMENTS.md

Reply:
There were originally but got removed https://github.com/opencontainers/wg-image-compatibility/pull/10#discussion_r1480255206